modules/assim.batch/R/pda.neff.R

```diff
@@ -0,0 +1,163 @@
+##' Autocorelation correction and efficient sample size calculation on latent process
+##'
+##' @title Calculate N_eff 
+##' @param inputs list
+##'
+##' @return inputs list, updated inputs with n_eff
+##'
+##' @author Istem Fer
+##' @export
+pda.neff.calc <- function(inputs){
```

Comment:
I know, all pda functions are dot separated :/ I need to sit and change them all at some point